Cython/Compiler/Code.py

```diff
@@ -644,17 +645,20 @@ class UtilityCode(UtilityCodeBase):
     file            filename of the utility code file this utility was loaded
                     from (or None)
     """
-    code_parts = ["proto", "impl", "init", "cleanup", "module_state_decls"]
+    code_parts = ["proto", "impl", "init", "cleanup", "module_state_decls", "module_state_traverse", "module_state_clear"]
 
     def __init__(self, proto=None, impl=None, init=None, cleanup=None,
-                 module_state_decls=None, requires=None,
+                 module_state_decls=None, module_state_traverse=None,
+                 module_state_clear=None, requires=None,
                  proto_block='utility_code_proto', name=None, file=None):
         # proto_block: Which code block to dump prototype in. See GlobalState.
         self.proto = proto
         self.impl = impl
         self.init = init
         self.cleanup = cleanup
         self.module_state_decls = module_state_decls
+        self.module_state_traverse = module_state_traverse
+        self.module_state_clear = module_state_clear
```

Comment:
These need to be added to `Code.pxd` as well.

Reply:
This is fixed in the split out "refactor" PR.